### PR TITLE
dashboard API data

### PR DESCRIPTION
**Pull request by anubra266**
Closes EveripediaNetwork/issues#569
CleanShot 2022-07-28 at 05 30 30@2x

### CODE REVIEW

File: src/pages/dashboard/index.tsx [hunk 1]
```diff
@@ -15,11 +15,162 @@ import {
...
+}
+
+enum GraphPeriod {
+  DAY = 'day',
+  WEEK = 'week',
+  MONTH = 'month',
+  YEAR = 'year',
+}
+
+const GRAPH_PERIODS = [
```
Comment (Emmanueldmlr) [inline src/pages/dashboard/index.tsx:83]: Don't you think we should define this in the data or constant folder?

**Comment by Emmanueldmlr** [comment]
image

having a margin at the bottom will be cooler.

**Review by Simplypheyie96 — COMMENTED** [review]
the bleed is absent on the dark mood. 
image

**Comment by anubra266** [comment]
> image
> 
> having a margin at the bottom will be cooler.

@Emmanueldmlr This concerns the general layout, so it's outside the scope of this PR, we can fix that in a different PR.

**Comment by Simplypheyie96** [comment]
> > image
> > having a margin at the bottom will be cooler.
> 
> @Emmanueldmlr This concerns the general layout, so it's outside the scope of this PR, we can fix that in a different PR.

i have added it to the dashboard fixes

**Review by Simplypheyie96 — APPROVED** [review]
l

**Review by Emmanueldmlr — APPROVED** [review]

**Pull request merged**